packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/mtom/cid.ts

```diff
@@ -1,9 +1,12 @@
 const cidPrefixRegex = /^cid:/;
-const endTagRegex = /^<|>$/g;
-export function stripCidPrefix(cid: string): string {
+function stripCidPrefix(cid: string): string {
   return cid.replace(cidPrefixRegex, "");
 }
 
-export function stripTags(content: string): string {
-  return content.replace(endTagRegex, "");
+function addTags(content: string): string {
+  return `<${content}>`;
+}
+
+export function getCidReference(cid: string): string {
```

Comment:
https://datatracker.ietf.org/doc/html/rfc2392

`A "cid" URL is converted to the corresponding Content-ID message header by removing the "cid:" prefix, converting the % encoded character to their equivalent US-ASCII characters, and enclosing the remaining parts with an angle bracket pair, "<" and ">".`

Reply:
this is from the RFC for content Ids in Multipart Messages